kroxylicious-operator/DEV_GUIDE.md

```diff
@@ -15,7 +15,7 @@ You should now be able to run the tests using `mvn`.
 If you want to run the `OperatorMain` (e.g. from your IDE, maybe for dubugging) then you'll need to install the CRD:
 
 ```bash
-kubectl apply -f src/main/resources/META-INF/fabric8
+kubectl apply -f ../kroxylicious-kubernetes-api//src/main/resources/META-INF/fabric8
```

Comment:
This gives me pause for thought. Do we want to pollute the top level with n operator modules?  I am wondering if now is the time to have a operator-parent?

Reply:
Maybe, but haven't built a hierarchy like that for anything else and I'd argue the fleet of KMS modules is a much bigger pollution. 

Its not something that personally bothers me so in the interests of focusing on higher value work I'm going to leave this PR as is. We can revisit later if we find a reason to do so.